tic_tac_toe/game.py

```diff
@@ -63,22 +92,53 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    rows = (
+        ((0, 0), (0, 1), (0, 2)),
+        ((1, 0), (1, 1), (1, 2)),
+        ((2, 0), (2, 1), (2, 2))
+    )
+
+    columns = (
+        ((0, 0), (1, 0), (2, 0)),
+        ((0, 1), (1, 1), (2, 1)),
+        ((0, 2), (1, 2), (2, 2))
+    )
+
+    diagonals = (
+        ((0, 0), (1, 1), (2, 2)),
+        ((0, 2), (1, 1), (2, 0))
+    )
+
+    winning_combinations = rows + columns + diagonals
```

Comment:
if there's some code to generate all the straight lines in the board, it would be awesome. we haven't figured out how so this is hard-coded, sadly.